<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search_Settings_Dlg</class>
 <widget class="QDialog" name="Search_Settings_Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>161</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>setting: where to search text by fields</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource2.qrc">
    <normaloff>:/view/img/btn-db/search-settings.svg</normaloff>:/view/img/btn-db/search-settings.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="dialogTopFrame">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>where to search : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmb_Fields"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_caseSensetive">
        <property name="minimumSize">
         <size>
          <width>344</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>case sensible</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_exactly">
        <property name="text">
         <string>exactly</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Search_Settings_Dlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Search_Settings_Dlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
